server/jetstream_test.go

```diff
@@ -8412,15 +8398,14 @@ func TestJetStreamDeleteMsg(t *testing.T) {
 				t.Fatalf("Unexpected error adding stream: %v", err)
 			}
 
-			nc := clientConnectToServer(t, s)
+			nc, js := jsClientConnect(t, s)
 			defer nc.Close()
 
 			pubTen := func() {
 				t.Helper()
 				for i := 0; i < 10; i++ {
-					nc.Publish("foo", []byte("Hello World!"))
+					js.Publish("foo", []byte("Hello World!"))
```

Comment:
use `sendStreamMsg` in all `js.Publish` new calls.

Reply:
Next time but agree.